src/client/mod.rs

```diff
@@ -288,6 +288,19 @@ impl Client {
         }
     }
 
+    pub(crate) async fn watch_helper(
```

Comment:
rather than calling this simply `watch_helper`, can we give it a more descriptive name? something like `start_change_stream` or something. Also, can we add a docstring for it? Doesn't have to be super long but it should explain why this method exists.

Reply:
fixed